0.6-dev/index.bs

```diff
@@ -0,0 +1,1787 @@
+    <td>Coordinate field transformation located at (path).
+  <tr><th>`inverseOf`
+    <td>`"transform":Transform`
+    <td>The inverse of a transformation. Useful if a transform is not closed-form invertible. See Forward and inverse for details and examples.
+  <tr><th>`bijection`
+    <td>`"forward":Transform`<br>`"inverse":Transform`
+    <td>Explicitly define an invertible transformation by providing a forward transformation and its inverse.
+  <tr><th>`byDimension`
+    <td>`"transformations":List[Transformation]`
+    <td>Define a high dimensional transformation using lower dimensional transformations on subsets of
+    dimensions.
+ <thead>
+   <tr><th>type<th>fields<th>description
+</table>
+
+
+Conforming readers:
+- MUST parse `identity`, `scale`, `translation` transformations;
+- SHOULD parse `mapAxis`, `affine` transformations;
+- SHOULD be able to apply transformations to points;
+- SHOULD be able to apply transformations to images;
+
+Coordinate transformations from array to physical coordinates MUST be stored in multiscales ([[#multiscale-md]]). 
+Transformations between different images MUST be stored in the attributes of a parent zarr group. For transformations that store
+data or parameters in a zarr array, those zarr arrays SHOULD be stored in a zarr group `"coordinateTransformations"`.
+
+<pre>
+store.zarr                      # Root folder of the zarr store
+│
+├── zarr.json                   # coordinate transformations describing the relationship between two image coordinate systems
+│                               # are stored in the attributes of their parent group.
+│                               # transformations between 'volume' and 'crop' coordinate systems are stored here.
+│
+├── coordinateTransformations   # transformations that use array storage go in a "coordinateTransformations" zarr group.
+│   └── displacements           # for example, a zarr array containing a displacement field
+│       └── zarr.json
+│
+├── volume
+│   ├── zarr.json               # group level attributes (multiscales)
+│   └── 0                       # a group containing the 0th scale
+│       └── image               # a zarr array
+│           └── zarr.json       # physical coordinate system and transformations here
+│                               # the array attributes
+└── crop
+    ├── .zattrs                 # group level attributes (multiscales)
+    └── 0                       # a group containing the 0th scale
+        └── image               # a zarr array
+            └── zarr.json       # physical coordinate system and transformations here
+                                # the array attributes
+</pre>
+
+### Additional details
+
+Most coordinate transformations MUST specify their input and output coordinate systems using `input` and `output` with a string value
+corresponding to the name of a coordinate system. The coordinate system's name may be the path to an array, and therefore may
```

Comment:
what should readers do if the `coordinateSystems` metadata contains a coordinate system with the same name as the path to the array?